crates/rpc/rpc-eth-api/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "reth-rpc-eth-api"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "Reth RPC `eth_` API implementation"
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+revm.workspace = true
+revm-inspectors = { workspace = true, features = ["js-tracer"] }
+revm-primitives = { workspace = true, features = ["dev"] }
+reth-errors.workspace = true
+reth-evm.workspace = true
+reth-metrics.workspace = true
+reth-network-api.workspace = true
+reth-primitives.workspace = true
+reth-provider.workspace = true
+reth-revm.workspace = true
+reth-rpc-server-types.workspace = true
+reth-rpc-types.workspace = true
+reth-rpc-types-compat.workspace = true
+reth-tasks = { workspace = true, features = ["rayon"] }
+reth-trie.workspace = true
```

Comment:
```suggestion
```

Reply:
used in `higher-ranked lifetime` error hack